File: serverApi/SRC/API/Model/PropostaAnexoModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using DOMAIN;
using Microsoft.AspNetCore.Http;
using Model;

namespace Model
{
    public class PropostaAnexoModel: EntidadeBase
    {
        public string Nome { get; set; }
        public string ContentType { get ; set;}
        public byte[] FileContent { get ; set;}
        public Proposta Proposta { get; set; }
        public IFormFile Anexo { get; set; }
    }
}

File: cliente/src/app/modules/propostas/model/proposta-anexo.model.ts

export class PropostaAnexoModel {
    constructor(
      public contentType?: string,
      public fileContent?: string,
      public file?: File,
      public nome?: string,
      public anexo?: string,
      public id?: string,
      public dataAtualizacao?: Date,
      public dataCriacao?: Date,
      public excluido?: boolean
    ){}
  }